app/views/audits/index.html.erb

```diff
@@ -26,7 +26,7 @@
           <td class="govuk-table__cell"> User deleted </td>
         <% end %>
         <td colspan="2" class="govuk-table__cell">
-          <% if audit_item.activity_type.include?("request") %>
+          <% if audit_item.activity_type.match(/validation|document_received_at_changed/) %>
```

Comment:
can we avoid a Regexp here and use `in?`

Reply:
in? only takes a single string, you'd instead need an iterator to pass through the two strings and check - which is much more logic then just having the two conditionals with an or - so the match is less code

Reply:
(I did try for a while to make in? work fyi, in light of your previous suggestion - but the problem is that we are checking if a substring is contained in audit_item, we aren't checking if a whole string is contained - ["document_received_at_changed", "validation"].in? audit_item.activity_type doesn't do the partial match of the word, hence the regex. Another option was to iterate with .any but to do that much to check two strings seemed too much overhead instead of just checking two conditions

Reply:
to be fair, does this need its own audit template at all? the generic one looks fine to me:

```ruby
  def audit_date_comment(document)
    previous, updated = document.saved_change_to_received_at


    { previous_received_date: previous.to_formatted_s(:day_month_year),
      updated_received_date: updated.to_formatted_s(:day_month_year)
    }.to_json
  end
```

![image](https://user-images.githubusercontent.com/107635/143438627-c3a96dfa-027e-4f8b-a743-6de5fb3abe9c.png)

Reply:
I did try that - its what I was saying - did you pull the code and try it? because when I did it it didn't pickup on the template with that exact code
```
audit.item_activity_type.in? ["document_received_at_changed", "request"]
```

Reply:
so what's the logic? whether it contains `request` or it's equal to `document_received_at_changed`?

Reply:
the logic is that in? doesn't take a partial string, so it can't match blahblah_request